src/lib/mongoBackend/MongoCommonUpdate.cpp

```diff
@@ -1619,21 +1643,21 @@ static bool processOnChangeConditionForUpdateContext
 }
 
 
-
+#if 0
```

Comment:
Yes, this was a test that worked and the function could be removed.
I forgot to completely remove it.